django_freeradius/settings.py

```diff
@@ -32,3 +32,4 @@
 SOCIAL_LOGIN_ENABLED = {'allauth.socialaccount',
                         'rest_framework.authtoken'}.issubset(settings.INSTALLED_APPS)
 DISPOSABLE_USER_TOKEN = getattr(settings, 'DJANGO_FREERADIUS_DISPOSABLE_USER_TOKEN', True)
+AUTOMATIC_GROUPNAME_ASSIGNMENT_RADIUSACCOUNTING = False
```

Comment:
this would make it always ``False`` regardless of configuration. the name is also too long, try the following:

```python
API_ACCOUNTING_AUTO_GROUP = getattr(settings, 'DJANGO_FREERADIUS_API_ACCOUNTING_AUTO_GROUP', True)
```
This also enables it by default, which is good because it should work in most cases and should not cause issues if enabled, but users may have their own reasons to disable it.

Reply:
Just to confirm, I would put the `DJANGO_FREERADIUS_API_ACCOUNTING_AUTO_GROUP` in the main settings file and `DJANGO_FREERADIUS_API_ACCOUNTING_AUTO_GROUP` in the app's settings file?